Test/git-issues/git-issue-356-errors.dfy

```diff
@@ -0,0 +1,226 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+
+// Check every possible conversion range error among int,char,bv,real,ORDINAL
+
+// int to char
+method Test1(n: int)
+{
+  var ch := n as char;
```

Comment:
When looking at just the `.dfy` test file, it's not clear where errors are produced or expected. I suggest following the convention of putting a comment that begins with the word "error" (typically `// error: ...`) in the `.dfy` file. This makes it easier to read. It also makes it easier to maintain when something changes.

Reply:
Marked the lines producing errors with "// ERROR". Methods that did not produce errors already have "OK" in their name.